ui/app/js/controllers/parent_controller.js

```diff
@@ -44,4 +44,8 @@ angular.module("app").config(['$httpProvider', function($httpProvider) {
 angular.module("app").controller('ParentController',
 function($scope, Page) {
   $scope.Page = Page;
+  $scope.isEmpty = isEmpty;
```

Comment:
Wow, I'm positively surprised we don't need to declare a dependency on `helpers.js`. Nice!